code/src/components/AddTodo.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
```

Comment:
I think you need to configure your Prettier to add semicolons for you because you are not consistent about it and it is should be automated 😸